docs/tutorialkit.dev/src/content/docs/reference/configuration.mdx

```diff
@@ -48,6 +48,13 @@ type I18nText = {
    */
   partTemplate?: string,
 
+  /**
+   * Text of the edit page link.
+   *
+   * @default 'Edit this page'
+   */
+  editPage?: string
```

Comment:
Nit: `editPage` -> `editPageText`.

The idea is:
 - When it ends with `Text` there are no interpolation, the text is copied verbatim.
 - When it ends with `Template`, string interpolation is possible.